app/react/Library/components/ExportButton.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Icon } from 'app/UI';
+import { t } from 'app/I18N';
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { wrapDispatch } from 'app/Multireducer';
+import { exportDocuments } from 'app/Library/actions/exportActions';
+import { ExportStore } from '../reducers/ExportStoreType';
+
+export type ExportButtonProps = {
+  processing: boolean;
+  storeKey: string;
+  exportDocuments: (keyStore: string) => any;
+};
+
+export class ExportButton extends Component<ExportButtonProps, {}> {
```

Comment:
Do not export the component if it is connected.  You can import the connected component in the test and either use the `.WrappedComponent` property of the component, or you can use '.dive()' methods to inject the store into the component.  That also allows you to test the mapStateToProps directly, which in this case are not tested at all.

You can check the dive method on `/app/react/components/Elements/specs/FeatureToggle.spec.tsx`